### DISCUSSION TITLE
Is there a way to reset keyboard shortcuts without reinstalling tabby?

### DISCUSSION THREAD

**Discussion by botruyens** [Eugeny/tabby#4490] [Q&A]
Thanks for the lovely useable product! Daily driver here.

I messed up the keyboard shortcut list a bit, and woulld love a away to reset them!

Thanks a bunch for the help!

Boris

**Comment by Eugeny** [comment]
Yes - just remove the `hotkeys` section from the config file (you can do this directly in `Settings -> Config file`)

**Reply by botruyens** [reply]
Awesome! Just what I needed, all defaults are shown here, I overlooked it. Thanks a bunch!
